avro-data/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -2127,16 +2127,12 @@ private String unionMemberFieldName(org.apache.avro.Schema schema,
   }
 
   private String unionMemberFieldName(Schema schema, boolean enhancedSchemaSupport) {
-    if (schema.type() == Schema.Type.STRUCT || isEnumSchema(schema)) {
+    if (schema.type() == Schema.Type.STRUCT || isEnumSchema(schema) || isFixedSchema(schema)) {
```

Comment:
Is there a existing test already covering FixedSchema with EnhancedSchemaSupport set or we need a new test?

Reply:
Yes a test already exists